### DISCUSSION TITLE
Why this dependency is not getting Release Notes in Renovate PRs?: @meltui/svelte

### DISCUSSION THREAD

**Discussion by oscard0m** [renovatebot/renovate#24304] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

PRs by Renovate bumping `@melt-ui/svelte` show the release notes.

Example PR: https://github.com/oscard0m/test-sveltekit/pull/11
Example Release I would expect to get the information extracted from: https://github.com/melt-ui/melt-ui/releases/tag/v0.47.0

### What you tried so far.

I would like to get more insight before debugging Renovate's code.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
It doesn't have release notes because the package does not include a `repository` entry in its published package.

From https://registry.npmjs.org/@melt-ui/svelte

<img width="509" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/c5e7b1c5-879e-4dfe-8cef-59bfa635a04f">

Compare that to https://registry.npmjs.org/@fontsource/fira-mono

<img width="573" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/1f3433f4-8ed7-4087-a6f4-b4cac7bc1055">

To address this you can contribute a PR to either the package's source itself (best) or to https://github.com/renovatebot/renovate/blob/0487578b339521ecc501f3ffab6c160d270c3786/lib/modules/datasource/metadata-manual.ts#L61 (workaround)

**Reply by oscard0m** [reply]
Thanks for pointing this out @rarkins and for being so fast (as always). I submitted a PR to add the `repository` property in `package.json`.

By reading the logs of Renovate, I was not able to catch this. Do you think this could be something to log as a `warning`? 

If you think it could be interesting, I'm happy to open an issue+PR.

**Reply by rarkins** [reply]
It's definitely not the right type of problem for a warning, because that creates a lot of noise for people for a problem which isn't their fault. If you were to look at the `packageFiles with updates` log message I think you'll see which dependencies have a sourceUrl and which don't. Potentially we could log a _debug_ messages, but the problem is that for many ecosystems (non-npm) it may be common and expected that it has no source URL.

**Reply by oscard0m** [reply]
> It's definitely not the right type of problem for a warning, because that creates a lot of noise for people for a problem which isn't their fault. If you were to look at the `packageFiles with updates` log message I think you'll see which dependencies have a sourceUrl and which don't. Potentially we could log a _debug_ messages, but the problem is that for many ecosystems (non-npm) it may be common and expected that it has no source URL.

I see your point. Maybe documenting it could be a nice addition?

**Reply by rarkins** [reply]
https://github.com/renovatebot/renovate/issues/24329
